docs/src/modules/components/Ad.tsx

```diff
@@ -198,7 +199,7 @@ export default function Ad() {
 
   React.useEffect(() => {
     // Avoid an exceed on the Google Analytics quotas.
-    if (Math.random() > GA_ADS_DISPLAY_RATIO || !eventLabel) {
+    if (Math.random() > ((process.env.GA_ADS_DISPLAY_RATIO ?? 0.1) as number) || !eventLabel) {
```

Comment:
One thing to note as well is that Next.js actually puts a number in `process.env.GA_ADS_DISPLAY_RATIO` while in Node.js this would always be a string, so the types will always be incorrect.
Personally, I'd never pass anything to a Next.js `env` that is not a string and always parse it from a string on the client side. This would avoid the `as number` which circumvents the type checking on this line.